src/core/attribute.d

```diff
@@ -0,0 +1,64 @@
+/**
+ * This module contains UDA's (User Defined Attributes) either used in
+ * the runtime or special UDA's recognized by compiler.
+ *
+ * Copyright: Copyright Jacob Carlborg 2015.
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Jacob Carlborg
+ * Source:    $(DRUNTIMESRC core/_attribute.d)
+ */
+
+/*          Copyright Jacob Carlborg 2015.
+ * Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
All other files are looking like that. I'm just being consistent. Personally I think it's redundant.